Mu2eG4/src/constructDiskCalorimeter.cc

```diff
@@ -264,11 +263,11 @@ namespace mu2e {
 
        //this is the full front panel
        G4Tubs* frontPlate = new G4Tubs("caloFrontPlate",FPInnerRadius,FPCoolPipeTorRadius+FPCoolPipeRadius,frontPanelHalfThick,0,CLHEP::twopi);       
-       G4LogicalVolume* frontPlateLog = caloBuildLogical(frontPlate, vacuumMaterial, "caloFrontPlateLog_"+std::to_string(idisk),0,G4Color::White(),0,0);
+       G4LogicalVolume* frontPlateLog = caloBuildLogical(frontPlate, vacuumMaterial, "caloFrontPlateLog",0,G4Color::White(),0,0);
```

Comment:
What is the motivation behind removing _idisk from here?

Reply:
Actually, please make sure all the physical volume names are unique, or one will not be able to retrieve/access them by name

Reply:
I meant physical and logical volume names

Reply:
I will add the disk id back in